spec/controllers/admin/rebate_forms_controller_spec.rb

```diff
@@ -101,4 +101,36 @@
 
     include_examples 'can wrangle rebate_forms'
   end
+
+  describe 'PUT update/:id' do
+    let(:user) { FactoryBot.create :user, council: rebate_form.property.council }
+    let(:valid_attributes) do
+      { fields: { valuation_id: "06601*004*02*", rates_bill: "1.10", dependants: "3", income_range: "below", lived_here_before_july_2018: "yes", full_name: "Sylvestor", email: "test@gmail.com", phone_number: "5556789", has_home_business: "no", email_phone_can_be_used: true, income: 23405.2, lived_with_partner: false } }
+    end
+
+    before do
+      sign_in user
+      put :update, params: { id: rebate_form.to_param, rebate_form: valid_attributes }
+    end
+
+    it 'should update updated_by column with current user' do
+      rebate_form.reload
+      expect(rebate_form.updated_by).to eq(user.id)
+    end
+
+    it 'should update full_name, dependants and income' do
+      rebate_form.reload
+      expect(rebate_form.fields['full_name']).to eq('Sylvestor')
+      expect(rebate_form.fields['dependants']).to eq('3')
+      expect(rebate_form.fields['income']).to eq('23405.2')
+    end
+
+    it 'should keep the same fields after an update' do
+      rebate_form.reload
+      rebate_form.fields.keys.each do |x|
+        expect(rebate_form.fields.include?(x)).to eq(true)
+      end
```

Comment:
You're checking that the fields are equal to the fields.
Can you check they're equal to what was there *before* the update?

e.g. you have this in the `let(:rebate_form):`
  phone_number: "5556789"

You can therefore do this: 
```
describe "preserves the existing fields" do
  it { expect(rebate_form.fields['email']).to eq "test@gmail.com" }
  it { expect(rebate_form.fields['phone_number']).to eq "5556789" }
  it { expect(rebate_form.fields['has_home_business']).to eq "no" }
  it { expect(rebate_form.fields['email_phone_can_be_used']).to eq true }
  it { expect(rebate_form.fields['lived_with_partner']).to eq false }
end
```

it's because we really need to know we're not gonna lose important data from production records when people edit it.